landscape/client/manager/scriptexecution.py

```diff
@@ -173,6 +175,8 @@ def _handle_execute_script(self, message):
         opid = message["operation-id"]
         try:
             user = message["username"]
+            if IS_SNAP:
+                user = "root"
```

Comment:
something about using `root` has really pissed off the github runners. I suggest an additional check for if we're running the tests and not doing this in that case.